03-task-manager/starter/app.js

```diff
@@ -3,12 +3,16 @@ const app = express();
 const tasks = require('./routes/tasks');
 const connectDB = require('./db/connect')
 require('dotenv').config();
```

Comment:
Nitpick:
I'd recommend moving this require('dotenv').config(); to the top of the file. If you were to have a file that invokes a function which uses an ENV, then on import/require I believe the ENV would be undefined since the import is before the .env is loaded.